microservices/domains-analytics-writer/dev/values.yaml

```
@@ -0,0 +1,19 @@
+name: "interop-be-domain-analytics-writer"
```

```suggestion
name: "interop-be-domains-analytics-writer"
```
double check name also with developers, this could be an annoying bug to catch in various resource definitions 😄